source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -645,7 +660,7 @@ function Get-SettingsFromResult
         $script:allResources = $DscResults.ResourcesNotInDesiredState + $DscResults.ResourcesInDesiredState
     }
 
-    return $script:allResources.Where({$PSItem.ResourceID -match $id})
+    return $script:allResources.Where( {$PSItem.ResourceID -match $id})
```

Comment:
adding the space here isn't needed, should remove the space between the "(" and the "{".

Reply:
Removed Whitespace